building/tracks/docs.md

```diff
@@ -10,7 +10,8 @@ docs
 ├── SNIPPET.txt (required)
 ├── REPRESENTER_NORMALIZATIONS.md (optional)
 ├── RESOURCES.md (required)
-└── TESTS.md (required)
+├── TESTS.md (optional)
+└── config.json (required)
```

Comment:
```suggestion
├── TESTS.md (required)
└── config.json (required)
```

They're both required

Reply:
So `config.json` is not required, but causes breakage down the line if it's not included? Seems odd to me, but not blocking so :man_shrugging: